src/cell.rs

```diff
@@ -75,97 +84,101 @@ pub fn range_manhattan4<T: Float>(p1: math::Point4<T>, p2: math::Point4<T>) -> T
 
 #[inline(always)]
 pub fn get_vec2<T: Float>(index: usize) -> math::Point2<T> {
-    let length = math::cast::<_,T>((index & 0xF8) >> 3) * math::cast(0.5 / 31.0);
+    let length = math::cast::<_, T>((index & 0xF8) >> 3) * math::cast(0.5 / 31.0);
     let diag = length * math::cast(0.70710678118);
     let one = length;
     let zero: T = math::cast(0.0);
     match index & 0x07 {
-        0 => [ diag,  diag],
-        1 => [ diag, -diag],
-        2 => [-diag,  diag],
+        0 => [diag, diag],
+        1 => [diag, -diag],
+        2 => [-diag, diag],
         3 => [-diag, -diag],
-        4 => [ one,   zero],
-        5 => [-one,   zero],
-        6 => [ zero,  one],
-        7 => [ zero, -one],
+        4 => [one, zero],
+        5 => [-one, zero],
+        6 => [zero, one],
+        7 => [zero, -one],
         _ => unreachable!(),
     }
 }
 
 #[inline(always)]
 pub fn get_vec3<T: Float>(index: usize) -> math::Point3<T> {
-    let length = math::cast::<_,T>((index & 0xE0) >> 5) * math::cast(0.5 / 7.0);
+    let length = math::cast::<_, T>((index & 0xE0) >> 5) * math::cast(0.5 / 7.0);
     let diag = length * math::cast(0.70710678118f32);
     let one = length;
     let zero = math::cast(0.0);
     match index % 18 {
-        0  => [ diag,  diag,  zero],
-        1  => [ diag, -diag,  zero],
-        2  => [-diag,  diag,  zero],
-        3  => [-diag, -diag,  zero],
-        4  => [ diag,  zero,  diag],
-        5  => [ diag,  zero, -diag],
-        6  => [-diag,  zero,  diag],
-        7  => [-diag,  zero, -diag],
-        8  => [ zero,  diag,  diag],
-        9  => [ zero,  diag, -diag],
-        10 => [ zero, -diag,  diag],
-        11 => [ zero, -diag, -diag],
-        12 => [ one,   zero,  zero],
-        13 => [ zero,  one,   zero],
-        14 => [ zero,  zero,  one],
-        15 => [-one,   zero,  zero],
-        16 => [ zero, -one,   zero],
-        17 => [ zero,  zero, -one],
+        0 => [diag, diag, zero],
+        1 => [diag, -diag, zero],
+        2 => [-diag, diag, zero],
+        3 => [-diag, -diag, zero],
+        4 => [diag, zero, diag],
+        5 => [diag, zero, -diag],
+        6 => [-diag, zero, diag],
+        7 => [-diag, zero, -diag],
+        8 => [zero, diag, diag],
+        9 => [zero, diag, -diag],
+        10 => [zero, -diag, diag],
+        11 => [zero, -diag, -diag],
+        12 => [one, zero, zero],
+        13 => [zero, one, zero],
+        14 => [zero, zero, one],
+        15 => [-one, zero, zero],
+        16 => [zero, -one, zero],
+        17 => [zero, zero, -one],
```

Comment:
Do you know of a way of disabling it for more 'grid-like' bits of code? An attribute?